src/plugins/data/common/search/search_source/inspect/inspector_stats.ts

```diff
@@ -25,17 +25,17 @@ export function getRequestInspectorStats(searchSource: ISearchSource) {
 
   if (index) {
     stats.indexPattern = {
-      label: i18n.translate('data.search.searchSource.indexPatternLabel', {
-        defaultMessage: 'Index pattern',
+      label: i18n.translate('data.search.searchSource.dataViewLabel', {
+        defaultMessage: 'Data view',
       }),
       value: index.title,
-      description: i18n.translate('data.search.searchSource.indexPatternDescription', {
-        defaultMessage: 'The index pattern that connected to the Elasticsearch indices.',
+      description: i18n.translate('data.search.searchSource.dataViewDescription', {
+        defaultMessage: 'The data view that connected to the Elasticsearch indices.',
```

Comment:
```suggestion
        defaultMessage: 'The data view that is connected to the Elasticsearch indices.',
```

Reply:
@lukasolson  what do you think? I see this as a change in tense so I want to make sure it makes sense with how search source works.

Reply:
This brings up a good point... I know we are renaming to distinguish an actual "index pattern" (the string) from the data view (the saved object). But what is actually shown here is just the index pattern string. What will we be showing for the title here?

![image](https://user-images.githubusercontent.com/1178348/130661337-320d634f-e09e-469a-90a2-e49ee697cef8.png)

I feel like whatever is shown in inspector should match that.

That being said, I am fine with either "The data view that was queried" or "The index pattern that was queried."

Reply:
Its currently the data view 'name' but someday may be the 'index pattern'. Unfortunately they're both the same thing right not.